packages/suite/src/components/suite/CollapsibleBox/index.tsx

```diff
@@ -162,15 +184,30 @@ const CollapsibleBox = ({
                 }}
             >
                 <Heading variant={variant}>{heading ?? iconLabel}</Heading>
-                <IconWrapper headerJustifyContent={headerJustifyContent}>
-                    {heading && iconLabel && <IconLabel>{iconLabel}</IconLabel>}
-                    <Icon
-                        icon="ARROW_DOWN"
-                        size={variant === 'tiny' ? 12 : 20}
-                        canAnimate={animatedIcon}
-                        isActive={!collapsed}
-                    />
-                </IconWrapper>
+                {withButton ? (
+                    <BoxCTA
```

Comment:
We don't want to have such a thing in universal reusable CollapsibleBox component. Some of the issues with it:
- reference to `document` in onClick (why is that even needed?)
- hardcoded text
- hardcoded size

Would be nice if there is a possibility to reuse some of the existing props (even if it slightly change the design) or at least pass it as a whole component from the Banner.

Reply:
reference to document.activeElement.blur() is a solution to unfocus button after clicking otherwise it keeps background color as active

So you suggest getting rid of a button and keep just an icon and probably text?

Reply:
I'm not suggesting anything, just to get rid of this feature specific component from shared CollapsibleBox component...